main.js

```diff
@@ -160,4 +160,136 @@ const inventory = [
     originalStock: 10,
     sold: 8,
   },
-];
\ No newline at end of file
+];
+
+// Opdracht 1 - Array Methoden
+// 1a
+const tvType = inventory.map((tv) => {return tv.type;});
```

Comment:
Wanneer je alles op één regel kunt schrijven, kun je het _return statement_ en de accolades weglaten. 
`const tvType = inventory.map((tv) => {return tv.type;})` wordt dan `const tvType = inventory.map((tv) => tv.type);`.
Dit geldt ook voor 1b, 1c en 1d. Is iets minder code.